subprojects/language-native/src/integTest/groovy/org/gradle/language/AbstractNativePreCompiledHeaderIntegrationTest.groovy

```diff
@@ -540,7 +540,7 @@ abstract class AbstractNativePreCompiledHeaderIntegrationTest extends AbstractIn
     }
 
     String getUniquePragmaOutput(String message) {
-        if (toolChain.displayName == "clang") {
+        if (toolChain.displayName.startsWith("clang")) {
```

Comment:
❓ Why was this changed?

Reply:
Because the displayName now shows up as "clang 9.1.0" or "clang 10.0.0" instead of just "clang".